dll/cpl/sysdm/lang/zh-TW.rc

```diff
@@ -286,17 +286,39 @@ BEGIN
     PUSHBUTTON "取消", IDCANCEL, 195, 224, 50, 14
 END
 
-IDD_EDIT_VARIABLE DIALOGEX 10, 15, 227, 71
-STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CONTEXTHELP | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
+IDD_EDIT_VARIABLE DIALOGEX 10, 15, 372, 71
+STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CENTER | WS_POPUPWINDOW | WS_CAPTION | WS_VISIBLE | WS_THICKFRAME
 CAPTION "編輯變數"
 FONT 9, "新細明體", 0, 0, 0x0
 BEGIN
+    SCROLLBAR IDC_DIALOG_GRIP, 365, 64, 7, 7, SBS_SIZEGRIP
     LTEXT "變數名稱(&N)︰", IDC_STATIC, 7, 14, 68, 8
     EDITTEXT IDC_VARIABLE_NAME, 75, 12, 145, 12, ES_AUTOHSCROLL
     LTEXT "變數值(&V)︰", IDC_STATIC, 7, 32, 68, 8
     EDITTEXT IDC_VARIABLE_VALUE, 75, 30, 145, 12, ES_AUTOHSCROLL
```

Comment:
Other languages (and zh-CN too) have the default dimensions of these elements changed... but not here?